tests/test.rs

```diff
@@ -59,29 +63,29 @@ fn test_c_return() {
     assert_eq!(2021, ffi::c_return_sum(2020, 1));
     match ffi::c_return_enum(0) {
         enm @ ffi::Enum::AVal => assert_eq!(0, enm.repr),
-        _ => assert!(false),
+        _ => unreachable!(),
```

Comment:
I think I would lean toward keeping this. The test is saying that reaching this codepath constitutes an assertion failure, not that reaching it is impossible.